frontend/src/components/ChatBox/ChatBoxInfoText.css

```diff
@@ -3,10 +3,19 @@
 	font-style: italic;
 	padding-top: 1rem;
 	text-align: center;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-top: 0.75rem;
+	padding: 0.5rem;
```

Comment:
Very minor, but I believe this `padding: 0.5rem` overrides the `padding-top`, so just remove the `padding-top` line.